kratos/python/add_utilities_to_python.cpp

```diff
@@ -106,6 +106,29 @@ void AddUtilitiesToPython()
     .def("SetVectorVar", &VariableUtils::SetVectorVar)
     .def("SetScalarVar", &VariableUtils::SetScalarVar< Variable<double> >)
     .def("SetScalarVar", &VariableUtils::SetScalarVar< VariableComponent< VectorComponentAdaptor<array_1d<double, 3> > > >)
+    .def("SetVectorVarNonHistorical", &VariableUtils::SetVectorVarNonHistorical)
+    .def("SetScalarVarNonHistorical", &VariableUtils::SetScalarVarNonHistorical< Variable<double> >)
+    .def("SetScalarVarNonHistorical", &VariableUtils::SetScalarVarNonHistorical< VariableComponent< VectorComponentAdaptor<array_1d<double, 3> > > >)
+    .def("SetVariable", &VariableUtils::SetVariable< bool >)
```

Comment:
Should the non historical values have a methods just called SetVariable?, and then a different name for the historical variables (just the nodes have historical variables)